py/server/deephaven/table.py

```diff
@@ -2087,9 +2087,10 @@ def slice(self, start: int, stop: int) -> Table:
     def slice_pct(self, start_pct: float, end_pct: float) -> Table:
         """Extracts a subset of a table by row percentages.
 
-        Returns a subset of table in the range [startPercentInclusive * sizeOfTable , endPercentExclusive * sizeOfTable).
+        Returns a subset of table in the range [start_pct * size_of_table , end_pct * size_of_table).
```

Comment:
I suggest including the floors so that super quant nerds and number lawyers know what is going on.